defaults/main.yml

```diff
@@ -34,43 +27,81 @@ faf_migrate_db: true
 # set up cron jobs for faf
 faf_cron_jobs: true
 
+# faf reachable on httpd root (http://hostname/)
+# if false it's (http://hostname/faf/)
+faf_web_on_root: false
+
+###########################
+# faf configuration       #
+###########################
+
+# [MAIN]
+faf_plugins_dir: /etc/faf/plugins/
+faf_templates_dir: /etc/faf/templates/
+faf_autoenableplugins: False
```

Comment:
Are these used anywhere?

Reply:
> Are these used anywhere?

Nice catch! They should be used in `etc-faf-faf.conf` template.

Reply:
Fixed with efbca26